front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -49,7 +58,15 @@ const useScenarioData = (scenario: ScenarioResponse, infra: InfraWithState) => {
     osrdEditoastApi.endpoints.getAllTimetableByIdTrainSchedules.useQuery(
       { timetableId: scenario?.timetable_id },
       {
-        skip: !scenario,
+        skip: !scenario || showPacedTrains,
```

Comment:
Hum I think you should fetch the paced trains even if we need to show the paced trains no ?
```suggestion
        skip: !scenario,
```

Reply:
What was I thinking 🙃